test/unit/murmurhash3_spec.js

```diff
@@ -0,0 +1,53 @@
+/* globals jasmine, expect, it, describe, MurmurHash3_64 */
+'use strict';
+
+describe('MurmurHash3_64', function() {
+
+  it('instantiates without seed ', function() {
+      var hash = new MurmurHash3_64();
+      expect(hash).toEqual(jasmine.any(MurmurHash3_64));
+  });
+  it('instantiates with seed ', function() {
+      var hash = new MurmurHash3_64(1);
+      expect(hash).toEqual(jasmine.any(MurmurHash3_64));
+  });
+
+  var hexDigestExpected = 'f61cfdbfdae0f65e';
+  var sourceText = 'test';
+  var sourceCharCodes = [116,101,115,116]; // 't','e','s','t'
+  it('generates expected from string', function() {
+      var hash = new MurmurHash3_64();
+      hash.update(sourceText);
+      expect(hash.hexdigest()).toEqual(hexDigestExpected);
+  });
+  it('generates expected from Uint8Array', function() {
+      var hash = new MurmurHash3_64();
+      hash.update(new Uint8Array(sourceCharCodes));
+      expect(hash.hexdigest()).toEqual(hexDigestExpected);
+  });
+  it('generates expected from Uint32Array', function() {
+      var hash = new MurmurHash3_64();
+      hash.update(new Uint32Array(sourceCharCodes));
+      expect(hash.hexdigest()).toEqual(hexDigestExpected);
+  });
+
+  it('hexdigest changes after update without seed', function() {
+      var hash = new MurmurHash3_64();
+      var hexdigest1, hexdigest2;
+      hash.update('test');
```

Comment:
Since you defined `var sourceText = 'test';` above, any reason not to just do `hash.update(sourceText);` here instead?
(This question applies in a couple of places below as well.)